services/QuillLMS/lib/benchmarks/diagnostic_insights/history_by_classroom.json

```diff
@@ -0,0 +1,275 @@
+{
+  "samples": 265,
+  "min_runtime": "84ms",
+  "mean_runtime": "1742ms",
+  "median_runtime": "702ms",
+  "ninety_fifth": "6475ms",
+  "max_runtime": "23925ms",
+  "data": {
```

Comment:
I don't know that recording the runtime for each individual classroom is particularly useful, but the data was already collected, so I figured I'd include it. That's all that's in the rest of this file, so doesn't need review.